core/src/test/scala/unit/kafka/admin/DeleteConsumerGroupsTest.scala

```diff
@@ -88,10 +88,12 @@ class DeleteConsumerGroupsTest extends ConsumerGroupCommandTest {
     val service = getConsumerGroupService(cgcArgs)
 
     TestUtils.waitUntilTrue(() => {
-      service.listGroups().contains(group)
+      service.collectGroupMembers(false)._2.get.size == 1
     }, "The group did not initialize as expected.", maxRetries = 3)
 
     val result = service.deleteGroups()
+    println(result)
```

Comment:
Unintentional line here?

Reply:
actually, quite intentional :) Those failures are difficult to reproduce, so more info in the console output of the test will help us next time.

I can add a comment if this is confusing.

Reply:
But this prints the output even when the test passes. If we want the output on failures only maybe that output can be added to the assertion error messages that follow. Unless there is another reason to have the output in either scenario?

Reply:
yeah, that makes sense. Will send the improvement tomorrow morning :)